src/Common/tests/System/PlatformDetection.cs

```diff
@@ -18,9 +18,10 @@ public static partial class PlatformDetection
         // do it in a way that failures don't cascade.
         //
 
-        public static bool IsUap => IsWinRT || IsNetNative;
+        public static bool IsUap => IsInAppContainer || IsNetNative;
```

Comment:
I just reread my comment which I linked above. I was suggesting the same as you are now. So I guess this was wrongly implemented? (this wasn't added in this PR, IsUap already exists in master...)